track_and_trace/server/scripts/seed_core_types.js

```diff
@@ -34,7 +34,22 @@ const types = require(`./${DATA}`)
 protos.compile()
   .then(awaitServerPubkey)
   .then(batcherPublicKey => getTxnCreator(null, batcherPublicKey))
+  .then(({createTxn, signer, signerPublicKey}) => {
```

Comment:
Where is this script used? We shouldn't be doing this stuff from the server anymore

Reply:
updated

Reply:
updated